docs/docs/04-standard-library/01-cloud/domain.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Domain
+id: domain
+description: A built-in resource for representing a domain in the cloud.
+keywords:
+  [
+    Wing reference,
+    Wing language,
+    language,
+    Wing standard library,
+    Wing programming language,
+    Domain,
+  ]
+sidebar_position: 1
+---
+
+The `cloud.Domain` resource represents the domain that will be configured when creating a website in the cloud.
```

Comment:
This should also be useful for other types of endpoints such as `cloud.Api`, not just websites.